README.md

```diff
@@ -1,9 +1,11 @@
 
 # Project Name
 
-MATH MAGICIANS
+
+REACT BOOKSTORE
```

Comment:
As stated by the previous reviewer, your styling is expected to match the wireframe as much as possible:
- [ ] You are missing the icon
- [ ] The categories menu should be closer to the books menu as we have on the wireframe
- [ ] Some the contents of the books should be aligned to the left as we have on the wireframe
- [ ] All books should be centered horizontally.

  ![init-sja](https://user-images.githubusercontent.com/54848363/180229164-34da8ff5-d4a0-4e7b-998e-5a36bb839ad3.jpg)